Library/Homebrew/test/caveats_spec.rb

```diff
@@ -26,4 +26,53 @@ def caveats
       expect(described_class.new(f)).not_to be_empty
     end
   end
+
+  describe "#caveats" do
+    context "when f.plist is not nil" do
+      it "prints plist startup information when f.plist_startup is not nil" do
+        p = formula do
```

Comment:
Using `f = formula do` works. There was some error earlier which got mistakenly corrected after I used another variable for the formula.